doc/index.md

```diff
@@ -1521,16 +1517,19 @@ Currently either [`"lua-zlib"`](https://github.com/brimworks/lua-zlib) or [`"lzl
 
 ### `inflate()` <!-- --> {#http.zlib.inflate}
 
-Returns a function that inflates (uncompresses) a zlib stream.
 
-The returned function takes a string of compressed data and an end of stream flag and returns the uncompressed data as a string. The function will throw an error if the stream is invalid. 
+Returns a reference to a function that inflates (uncompresses) a zlib stream.
```

Comment:
It's not a reference to a function, it's a function (infact it's a closure if you want to be more precise).